src/Microsoft.Diagnostics.Monitoring.EventPipe/Logs/EventLogsPipelineSettings.cs

```diff
@@ -3,17 +3,19 @@
 // See the LICENSE file in the project root for more information.
 
 using Microsoft.Extensions.Logging;
-using System;
 using System.Collections.Generic;
-using System.Text;
 
 namespace Microsoft.Diagnostics.Monitoring.EventPipe
 {
     internal class EventLogsPipelineSettings : EventSourcePipelineSettings
     {
-        public LogLevel LogLevel { get; set; }
+        // The default log level for all categories
+        public LogLevel LogLevel { get; set; } = LogLevel.Trace;
 
-        //This setting will set the levels to application default.
-        public bool UseAppFilters { get; set; }
+        // The logger categories and levels at which log entries are collected.
+        public IDictionary<string, LogLevel?> FilterSpecs { get; set; }
+
+        // This setting will collect logs for the application-defined categories and levels.
+        public bool UseAppFilters { get; set; } = false;
```

Comment:
I believe at some point the UseAppFilters will get backported to 3.1. At that point it would probably make sense to make true the default here.

Reply:
AFAIK the backport already got merged and is shipping with May servicing release which should be out in a few days.

Reply:
Yeah, it was released yesterday. I will update the default to true. But this will require a small change in dotnet-monitor to explicitly disable UseAppFilters in the case the user specifies a log level in the query string of the GET /logs route.

Reply:
I changed the default to true.